libs/service-portal/core/src/components/ActionCard/ActionCard.tsx

```diff
@@ -37,7 +46,9 @@ export const ActionCard: FC<Props> = ({ label, title, date, cta }) => {
           justifyContent="spaceBetween"
           alignItems="center"
         >
-          <Text variant="h4">{title}</Text>
+          <Box onClick={cta.onClick} cursor="pointer">
```

Comment:
This should probably be a button, and have a blue color to indicate that it's clickable